Sources/BluetoothServices/Characteristics/TemperatureType.swift

```diff
@@ -0,0 +1,53 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import NIO
+import SpeziBluetooth
+
+
+/// The location of a temperature measurement.
+public enum TemperatureType: UInt8, CaseIterable {
```

Comment:
If this is from some sort of standard it would be gerat if we could link to it in the documentation.